gradle.properties

```diff
@@ -1,10 +1,10 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=1.1.0-alpha5-SNAPSHOT
-ballerinaLangVersion=2.0.0-alpha5
-stdlibIoVersion=0.6.0-alpha4
-stdlibOsVersion=0.8.0-alpha4
-stdlibRegexVersion=0.7.0-alpha4
+ballerinaLangVersion=2.0.0-alpha6-SNAPSHOT
+stdlibIoVersion=0.6.0-alpha5-SNAPSHOT
+stdlibOsVersion=0.8.0-alpha5-SNAPSHOT
+stdlibRegexVersion=0.7.0-alpha4-SNAPSHOT
```

Comment:
Is this regex version correct?

Reply:
This is updated in https://github.com/ballerina-platform/module-ballerina-log/pull/79